### PR TITLE
#2156 Remove KSM from Spotlight & Series insight

### LINKED ISSUE #2156: Remove KSM from Spotlight & Series insight
Hence we have right now support only KSM, apply DRY method and remove repeating stuff which doesn't help you at all just noise the dashboard

Remove KSM from columns where it repeats :)

image

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2156
- [x] Made another PR, I misread the task previously
thanks @roiLeo for correcting it. 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot 2022-02-01 at 9 16 06 PM
Screenshot 2022-02-01 at 9 16 09 PM

### CODE REVIEW

**Comment by yangwao** [comment]
I guess would add rounding to two decimals in spotlight 
image

Same with series and we are good to go! :) 

image

File: components/series/SeriesTable.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@
         numeric
         cell-class="is-vcentered">
         <template v-if="!isLoading">
-          <Money :value="props.row.volume" inline />
+          <Money :value="props.row.volume" inline :hideUnit="true" />
```
Comment (roiLeo) [inline components/series/SeriesTable.vue:93]:
```suggestion
          <Money :value="props.row.volume" inline hideUnit />
```

File: components/series/SeriesTable.vue [hunk 2]
```diff
@@ -184,7 +184,7 @@
         cell-class="is-vcentered"
         sortable>
         <template v-if="!isLoading">
-          <Money :value="props.row.floorPrice" inline />
+          <Money :value="props.row.floorPrice" inline :hideUnit="true" />
```
Comment (roiLeo) [inline components/series/SeriesTable.vue:187]:
```suggestion
          <Money :value="props.row.floorPrice" inline hideUnit />
```

File: components/spotlight/SpotlightTable.vue [hunk 3]
```diff
@@ -88,7 +88,7 @@
         v-slot="props"
         sortable>
         <template v-if="!isLoading"
-          ><Money :value="props.row.averagePrice" inline
+          ><Money :value="props.row.averagePrice" inline :hideUnit="true"
```
Comment (roiLeo) [inline components/spotlight/SpotlightTable.vue:91]:
```suggestion
          ><Money :value="props.row.averagePrice" inline hideUnit
```

File: components/spotlight/SpotlightTable.vue [hunk 4]
```diff
@@ -103,7 +103,7 @@
 
       <b-table-column field="volume" label="Volume" v-slot="props" sortable>
         <template v-if="!isLoading"
-          ><Money :value="props.row.volume" inline
+          ><Money :value="props.row.volume" inline :hideUnit="true"
```
Comment (roiLeo) [inline components/spotlight/SpotlightTable.vue:106]:
```suggestion
          ><Money :value="props.row.volume" inline hideUnit
```

**Review by roiLeo — COMMENTED** [review]
for more consistency

**Comment by yangwao** [comment]
> I guess would add rounding to two decimals in spotlight

I guess would be amazing to add rounding to two decimals in spotlight and series and we are good to go! :)

**Comment by prachi00** [comment]
@yangwao done
Screenshot 2022-02-03 at 3 20 59 PM

Screenshot 2022-02-03 at 3 20 56 PM

**Comment by yangwao** [comment]
Why are series insight first rows 0.00 and Nan? 

Side by side your deploy preview vs beta `main` branch

image

Seems spotlight is same 

image

**Comment by prachi00** [comment]
Screenshot 2022-02-03 at 4 07 55 PM
Screenshot 2022-02-03 at 4 08 02 PM

@yangwao the issue was with numbers that had comma, like **15,382** , javascript's toFixed function doesn't parse strings, so I had to remove the comma manually. So while it works fine now, there is currently no comma on volume column.

Let me know if you want to add it, then I'll write another function that adds comma manually to numbers

**Comment by yangwao** [comment]
I see, I guess it's good for now :)

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 159.47 USD/KSM ~ 0.627 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x6377cc195a5d8cd78a0874bf3931ce8d42c06edc49a94a66b5219a276d6ca5e0](https://kusama.subscan.io/extrinsic/0x6377cc195a5d8cd78a0874bf3931ce8d42c06edc49a94a66b5219a276d6ca5e0)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**